src/command-line/index.js

```diff
@@ -23,7 +23,7 @@ if (program.home) {
 
 var config = Helper.HOME + "/config.js";
 if (!fs.existsSync(config)) {
-	mkdirp.sync(Helper.HOME);
+	mkdirp.sync(Helper.HOME, {mode: 0o0700});
```

Comment:
Could you use `'0700'` instead please?
ESLint is not too happy with this and octal mode is not allowed in strict mode (not that we enforce it at the moment, but it would be nice as it's a simple addition and helps finding out issues...).